components/image/image.css

```diff
@@ -0,0 +1,13 @@
+@import url("../../styles.css");
+
+
+.img-responsive {
+    width: 100%; 
+    height: auto;
+    object-fit: cover;
+    object-position: center;
+}
+
+.img-round {
+    border-radius: 50%;
+}
```

Comment:
add equal height and width or look into aspect-ratio property to make image actually round.

Reply:
Updated the class. Thank you